assembly/integer/u256.ts

```diff
@@ -386,33 +387,87 @@ export class u256 {
     return new u256(a.lo1 & b.lo1, a.lo2 & b.lo2, a.hi1 & b.hi1, a.hi2 & b.hi2);
   }
 
+  /**
+   * Right shifts a 256-bit number by a specified number of bits.
+   * 
+   * @param value - The 256-bit number to be shifted.
+   * @param shift - The number of bits to shift.
+   * @returns A new u256 value that represents the shifted number.
+   * 
+   * @remarks
+   * When the shift value is >= 256, the result is always zero. 
+   * For shifts < 0, the function returns the original value without any shifts.
+   */
   @operator('>>')
   static shr(value: u256, shift: i32): u256 {
-    shift &= 255;
-    var off = shift as u64;
-    if (shift <= 64) {
-      if (shift == 0) return value;
-      let hi2 =  value.hi2 >> off;
-      let hi1 = (value.hi1 >> off) | (value.hi2 << 64 - off);
-      let lo2 = (value.lo2 >> off) | (value.hi1 << 64 - off);
-      let lo1 = (value.lo1 >> off) | (value.lo2 << 64 - off);
-      return new u256(lo1, lo2, hi1, hi2);
-    } else if (shift > 64 && shift <= 128) {
-      let hi1 = value.hi2 >> 128 - off;
-      return new u256(value.lo2, value.hi1, hi1);
-    } else if (shift > 128 && shift <= 192) {
-      let lo2 = value.hi2 >> 192 - off;
-      return new u256(value.hi1, lo2);
-    } else {
-      return new u256(value.hi2 >> 256 - off);
+    if (shift >= 256) return u256.Zero;
+    if (shift <= 0) return value;
+
+    if(shift > 128) {
+      let low = new u128(value.hi1, value.hi2) >> (shift - 128);
+      return new u256(low.lo, low.hi, 0, 0);
```

Comment:
This is mismatch behavior of `u128.shr`. We don't use such safe checks for usual u128 / u256. And use wrapping behavior which is expected for all basic shifts including (u32, i32, u64, i64). Safe shifting ops uses only in `safe/u128`, `safe/u256` and etc in this namespace: https://github.com/MaxGraey/as-bignum/tree/master/assembly/integer/safe

So I propose delete this chages. Also doesn't relate to div / rem